dev-system/dev/user.clj

```diff
@@ -323,8 +310,9 @@
   (sit-sys/stop)
   ; Stops the running code
   (stop)
+  ;; TODO: Generic work: Delete the next 2 lines to get rid of (distribute-schemas).
```

Comment:
Make this a sub-task or checklist item on the parent ticket.

Reply:
I have deleted this code and comment.